src/TextInput/TextInput.svelte

```diff
@@ -1,50 +1,51 @@
 <script>
   /**
-   * @event {null | number | string} change
-   * @event {null | number | string} input
```

Comment:
Such changes will certainly break existing codebases. 

I think we should adopt v11 styles primarily while supporting existing codebases as much as possible.

Once v11 styles are merged to master we can then work towards feature parity with the canonical implementation (React). 

V11 styles will be even more delayed if we try to achieve multiple goals at the same time.

Reply:
This is something that will happen eventually anyway. I'd make the case that given we're already making breaking changes, it may as well all come at once.

Most of the fixes to codebases will be relatively trivial (i.e. largely find & replace and copy-paste).

Reply:
I’m glad you’re pushing this project forward. We want the same thing. My worry is that once we open a PR from [next/v11-styles → master](https://github.com/carbon-design-system/carbon-components-svelte/compare/master...next/v11-styles) the reviewer, i.e. @metonym, will have a very hard time assessing all these to some extend unnecessary changes. What I’m saying is that we will have v11 styles on the `master` branch earlier if we keep the change to a minimum.

> This is something that will happen eventually anyway.

We could propose this very change to `master` then. It’s unrelated to v11 styles. Feature parity with the canonical implementation is a separate issue.

> I'd make the case that given we're already making breaking changes, it may as well all come at once.

1. I don’t think we should make breaking changes. At least I tried [not to make](https://github.com/carbon-design-system/carbon-components-svelte/pull/1709/files#diff-49e1f6991bcb2a3f21420920a5e8e3c3b9760c9d316c50d5390779ee018ddb69R16) any unnecessary breaking changes.
2. I don’t think making them all at once is a good idea because it makes reviewing and writing release notes very hard.

> Most of the fixes to codebases will be relatively trivial (i.e. largely find & replace and copy-paste).

Are you running a `carbon-components-svelte` app in production? We do. If we suddenly miss events on `TextInput` we cannot just run find & replace. It’ll break the application.

Reply:
I appreciate the discussion. I hope I have addressed Gregor's concerns in https://github.com/carbon-design-system/carbon-components-svelte/discussions/1872#discussioncomment-8185572 - let's continue there.